### ISSUE TITLE
Inconsistent column order in reports

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#1611]
Each tab of the reports page has a different column order.

On the test plan tab, the order is:
1. JAWS and Chrome
2. NVDA and Chrome
3. VoiceOver for macOS and Safari

On the ARIA features tab, the order is:
1. VoiceOver for macOS and Safari
2. JAWS and Chrome
3. NVDA and Chrome

On the HTML feature tab, the order is:
1. VoiceOver for macOS and Safari
2. NVDA and Chrome
3. JAWS and Chrome

I propose the order should be alphabetical first by screen reader then by browser, so the order would be ...
1. JAWS and Chrome
2. JAWS and Edge
3. JAWS and Firefox
4. NVDA and Chrome
5. NVDA and Edge
6. ...
